dataframe-geo/src/main/kotlin/org/jetbrains/kotlinx/dataframe/geo/geotools/ToGeoDataFrame.kt

```diff
@@ -0,0 +1,56 @@
+package org.jetbrains.kotlinx.dataframe.geo.geotools
+
+import org.geotools.api.feature.simple.SimpleFeature
+import org.geotools.api.feature.simple.SimpleFeatureType
+import org.geotools.api.feature.type.GeometryDescriptor
+import org.geotools.api.referencing.crs.CoordinateReferenceSystem
+import org.geotools.data.simple.SimpleFeatureCollection
+import org.jetbrains.kotlinx.dataframe.DataColumn
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.api.Infer
+import org.jetbrains.kotlinx.dataframe.api.toDataFrame
+import org.jetbrains.kotlinx.dataframe.geo.GeoDataFrame
+import org.jetbrains.kotlinx.dataframe.geo.GeoFrame
+import org.locationtech.jts.geom.Geometry
+
+fun SimpleFeatureCollection.toGeoDataFrame(): GeoDataFrame<*> {
```

Comment:
please rename the file to "toGeoDataFrame" like you did with the other file

Reply:
Done